content/docs/roadmap/v1.6.0.md

```diff
@@ -0,0 +1,108 @@
++++
+title = "v1.6.0"
+weight = 1
++++
+
+# NSM v1.5.0
+
+## Release Schedule
+
+- **Feature Freeze** -  2022-09-05
+- **Release Candidate** -  2022-09-12
+- **Release Notes** -  2022-09-12
+- **Release** -  2022-09-19
+
+
+## Planned Enhancements
+
+
+### Update NSM sdk/api dependencies
+
+
+Planned to update next sdk/api dependencies:
+
+- Open Polic Agent
+- OpenTelemetry
+- gRPC
+- protobuf
+- JWT
+
+To keep NSM uptodate.
+
+
```

Comment:
This is definitely a thing to do for v1.6, but let's keep the roadmap to new software features/improvements. :)

Reply:
Sure. Fixed.